.github/actions/get_repo_name/action.yml

```diff
@@ -0,0 +1,21 @@
+name: 'Get Repo Name'
+description: 'Gets the repository name that was added to the open-repositories.txt file.'
+outputs:
+  repo-name:
+    value: ${{ steps.get-repo-name.outputs.new_repo }}
+    description: "The name of the repository that was added."
+
+runs:
+  using: 'composite'
+  steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Get repo name
+        id: get-repo-name
+        run: |
+          NEW_REPO=$(git diff origin/master...HEAD open-repositories.txt | grep "+" | tail -n 1 | sed 's/+//')
```

Comment:
I guess this will silently ignore all but the last repo if multiple added in the single PR.
The simples option could be to check and reject PR trying to add multiple repos.

Reply:
Murphy's law says you can't :)

Reply:
Okay I think my update works - I added another test here: https://github.com/dfinity/repositories-open-to-contributions/pull/35